apps/web/src/server/applications/case/examination-timetable/applications-timetable.service.js

```diff
@@ -15,8 +17,8 @@ export const getCaseTimetableItemTypes = async () => {
 /**
  * Save new timetable item
  *
- * @param {ApplicationsTimetable} payload
- * @returns {Promise<{updatedTimetable?: ApplicationsTimetable, errors?: import('@pins/express').ValidationErrors}>}
+ * @param {ApplicationsTimetablePayload} payload
+ * @returns {Promise<{updatedTimetable?: ApplicationsTimetable, errors?: ValidationErrors}>}
  */
 export const createCaseTimetableItem = async (payload) => {
```

Comment:
this and the function below seem exactly the same, can we use one for both create and update?

Reply:
they're slightly different: one points to patch('...timetable/$id)' and the other one to post('timetable/').

I can unify them but it's probably cleaner this way.